src/assert.js

```diff
@@ -0,0 +1,5 @@
+export default function assert(condition, message) {
+  if (!condition) {
+    throw new Error(`Assertion failed. ${message}`);
```

Comment:
Just `message`? In a debugging context one can already see that it comes from assert()

Reply:
Was emulating the standard assert method. You have a point that since our coding guideline is to not use assert messages, there is no need to support it here. Updated.